README.md

```
@@ -9,11 +19,15 @@ A minimal, efficient tool for downloading global air quality data from OpenAQ. D
 pip install -r requirements.txt
 ```
 
-2. Get your OpenAQ API key(s):
+2. Configure API keys:
+   **For OpenAQ (air quality data):**
    - Sign up at https://explore.openaq.org/register
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)